schema/caveschema.xsd

```diff
@@ -29,10 +28,109 @@
             </xs:sequence>
           </xs:complexType>
         </xs:element>
-        <xs:element name="PlacementRoot" minOccurs="0" maxOccurs="1">
+        <xs:element name="PlacementRoot" maxOccurs="1">
```

Comment:
My linter says that `maxOccurs="1"` is redundant
[w3schools](https://www.w3schools.com/xml/schema_complex_indicators.asp) seems to concur in section "Occurrence Indicators":

> Note: For all "Order" and "Group" indicators (any, all, choice, sequence, group name, and group reference) the default value for maxOccurs and minOccurs is 1.



```suggestion
        <xs:element name="PlacementRoot">
```

Reply:
Did some local testing and it's definitely true, will delete all of the `maxOccurs="1"`